fix/upp/postxconfig-NT-fv3lam_rrfs.txt

```diff
@@ -1127,43 +1127,6 @@ isobaric_sfc
 ?
 ?
 ?
-153
```

Comment:
@EricJames-NOAA  It looks like you removed all CLMR fields from the control file?

Reply:
Oh man how did I do that.  Let me try again.